app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+class CommandError(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3:
+        raise CommandError(f"Wrong format of command: {command}")
+
+    if command[0] != "mv":
```

Comment:
There is a better way to write a condition: 
 - do not change the incoming string and write down: if command.startswith("mv ")

Reply:
`command` already split in parts and I need it splitted to unpack other parts of command later in code, your suggestion will break code logic because `command` already is list object which has no `startswith ()` method and also this will add unnecessary check before splitting. Why do I need to do this if I will later split it still?